azure/README.md

```diff
@@ -16,15 +15,20 @@ az group create --name "${RESOURCE_GROUP}" --location "${LOCATION}"
 ```bash
 SUBSCRIPTION_ID=$(az account show --query id --output tsv)
 
-az ad sp create-for-rbac \
-  --role Contributor \
-  --scopes "/subscriptions/$SUBSCRIPTION_ID" \
-  --query "{ client_id: appId, client_secret: password, tenant_id: tenant }"
+az ad sp create-for-rbac
+  --name "Packer Build"
+  --role "Contributor"
+  --scopes /subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP}
+
+az role assignment create
+  --assignee ${CLIENT_ID}
+  --role "Contributor"
+  --scope /subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP}
```

Comment:
```suggestion
az ad sp create-for-rbac \
    --name "Packer Build" \
    --role "Contributor" \
    --scopes /subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP} \
    --query "{ CLIENT_ID: appId, CLIENT_SECRET: password, TENANT_ID: tenant }"

az role assignment create \
  --assignee ${CLIENT_ID} \
  --role "Contributor" \
  --scope /subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP}
```

Reply:
Also the last command fails with the following error:

```bash
$ az role assignment create \
  --assignee ${CLIENT_ID} \
  --role "Contributor" \
  --scope /subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP}
argument --assignee: expected one argument
```

My guess is that `CLIENT_ID` is not populated, the new doc should mention where this `CLIENT_ID` is coming from.

Reply:
I think I will divide the above two commands with instructions between them to export respective environment variable.